justfile

```diff
@@ -12,3 +12,10 @@ build-push-container IMAGE:
     ./scripts/movement/build-push-image {{ IMAGE }}
 container-tests:
     ./scripts/tests/container-tests
+
+# E2E Tests for GGP Deprecation
+test-e2e-verify-collect-fee:
+    process-compose -f process-compose/movement-full-node/process-compose.test-e2e-verify-collect-fee.yml up --wait --follow
+
+test-e2e-framework-upgrade-collect-gas-fees:
+    process-compose -f process-compose/movement-full-node/process-compose.test-e2e-framework-upgrade-collect-gas-fees.yml up --wait --follow
```

Comment:
You can remove this. E2E tests require a full-node setup and faucet. We are probably going to run it with
```
 just movement-full-node native build.setup.test-e2e-verify-collect-fee
 just movement-full-node native build.setup.
test-e2e-framework-upgrade-collect-gas-fees
```